json/src/hash.rs

```diff
@@ -19,7 +19,7 @@
 use std::str::FromStr;
 use serde::{Deserialize, Deserializer, Error};
 use serde::de::Visitor;
-use util::hash::{H64 as Hash64, Address as Hash160, H256 as Hash256, H2048 as Hash2048};
+use util::hash::{H64 as Hash64, H160 as Hash160, H256 as Hash256, H2048 as Hash2048};
```

Comment:
prefer to stick to `Address` over `H160` as long as it's being used for an address.